browser/js/films/FilmFactory.js

```diff
@@ -1,22 +1,15 @@
 'use strict';
-
 app.factory('FilmFactory', function ($http) {
 	return {
-		getFilms: function(){
-			return $http.get('/api/films').then(function (response) {
+		getFilms: function(category){
+			return $http.get('/api/films', { 
+                // params: {categories: "5532c18a0dad5827ab24af49"}
+                params: {categories: category}
+                // console.log("PARAMS:",params);
```

Comment:
Could we please get rid of comment before we commit in the future? Would just make things easier to read. Thanks!